create-snowpack-app/app-template-minimal/index.js

```diff
@@ -0,0 +1,2 @@
+/* Add JavaScript code here! */
+console.log('Hello World!');
```

Comment:
nit: lets add a bit of personality here :) ex: "A wild console log message appears!", "script connected successfully", etc.